### ISSUE TITLE
Dirty option for clickable

### ISSUE THREAD

**Issue by kugiigi** [milikhin/seabass2#124]
**Is your feature request related to a problem? Please describe.**
There are times you don't want or need to recompile C++ parts and the dirty option in clickable can be helpful in this cases.

**Describe the solution you'd like**
An option to build with the dirty option.

**Describe alternatives you've considered**
Just build C++ parts all the time 😄

**Comment by mattbel10** [comment]
I agree with @kugiigi request, but not only in regard to the `dirty` option, I would extend his request to all the extensions `clickable` gives the chance to exploit: for instance I'd also need a `ssh` specifcation with my custom IP address and also a ` build-libs` command to compile libraries prior to compile the app itself. What do you think to provide a text file in the Seabass2 app folder, where we might write down the custom command that the Seabass2 embedded build button would read from? That would give a great flexibility. I just started to use this app since a week now but the more I use it, the more I like it. For sure I'll provide soon a bounty to @milikhin ;)
